### PR TITLE
Add crypto_pkg

### PR DESCRIPTION
**Pull request by programmingAthlete**
add crypto_pkg cryptanalysis package

### CODE REVIEW

**Review by devilish66694 — APPROVED** [review]

**Review by Astrotony — APPROVED** [review]

**Review by MarkjhuneBahian — CHANGES_REQUESTED** [review]
Iyotabe

**Review by alfredmang — APPROVED** [review]

**Comment by merlinwizzard** [comment]
{[Remove-\-Damon-\Reinstall-\Main-Repositiory-\-]}{[Run-Prg]}

**Review by Kuntree12 — APPROVED** [review]

**Review by alfredmang — COMMENTED** [review]
Fake online login

**Review by alfredmang — CHANGES_REQUESTED** [review]
Deny access

**Comment by programmingAthlete** [comment]
> Fake online login

@alfredmang, what do you mean?

**Review by alfredmang — APPROVED** [review]

**Review by alfredmang — APPROVED** [review]
add crypto_pkg cryptanalysis package

**Review by Kuntree12 — APPROVED** [review]

**Review by Jitendrapande20 — APPROVED** [review]

**Review by Amaryasir9 — COMMENTED** [review]
Good

**Review by Katka1983 — APPROVED** [review]

**Comment by programmingAthlete** [comment]
Need the approval of someone with writing access. If you have writing access and you agree with this PR, please approve it

**Review by devilish66694 — APPROVED** [review]

**Comment by Emmanuel001v** [comment]
Nice

**Review by devilish66694 — APPROVED** [review]

**Review by devilish66694 — APPROVED** [review]

**Review by Moneyman60 — APPROVED** [review]

**Review by Moneyman60 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -67,6 +67,7 @@ Repository | Description
 [APT Notes](https://github.com/kbandla/APTnotes) 									| Various public documents, whitepapers and articles about APT campaigns
 [Bug Bounty Reference](https://github.com/ngalongc/bug-bounty-reference) 			| List of bug bounty write-up that is categorized by the bug nature
 [Cryptography](https://github.com/sobolevn/awesome-cryptography) | Cryptography resources and tools
+[CryptoPkg](https://github.com/programmingAthlete/crypto_pkg) | Cryptanalysis package containing attacks to modified versions of some known symmetric key ciphers and correlation power analysis attack on full AES
```
Comment (Moneyman60) [inline README.md:70]: Yes

**Review by Moneyman60 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**